src/components/show-products/index.js

```diff
@@ -132,15 +144,22 @@ export const ShowProducts = () => {
                   aria-label={TimeFrames[timeFrame]}
                 />
                 <span
-                  className={`checkmark checkbox-timeFrame-${timeFrame}`}
+                  className={`checkmark checkbox - timeFrame - ${timeFrame} `}
```

Comment:
I guess there are some extra whitespaces. Maybe, you forgot to remove those whitespaces 😄 ! Because, I couldn't see your amazing colors for soon, kind of soon, and so on.

**Current state**
![image](https://user-images.githubusercontent.com/8689897/141539206-9abd0a44-1205-4c45-887d-8e90b524d192.png)

**Possible expected result**

![image](https://user-images.githubusercontent.com/8689897/141539275-2fc17608-2668-47bf-95bf-465bbadca791.png)





```suggestion
                  className={`checkmark checkbox-timeFrame-${timeFrame}`}
```